src/applications/disability-benefits/all-claims/config/781/index.js

```diff
@@ -0,0 +1,44 @@
+import { incidentDate, secondaryIncidentDate } from '../../pages';
+
+import { needsToEnter781, needsToEnter781a } from '../../utils';
+
+export function formConfig781(iterations) {
+  let configObj = {};
+  for (let index = 0; index < iterations; index++) {
+    configObj = {
+      ...configObj,
+      // 781 PAGE CONFIGS GO HERE
+      ...{
+        [`incidentDate${index}`]: {
+          title: 'PTSD Incident Date',
```

Comment:
Have you tested what this looks like on the review page? Might need to distinguish the titles else it could look strange there.

Reply:
Maybe something like this?

```
...{
  [`incidentDate${index}`]: {
    title: `PTSD Incident ${index} Date`,
    ...
  }
...
}
```

Reply:
This part of the review page currently looks like this.
![image](https://user-images.githubusercontent.com/12970166/48577247-8c05de00-e8cb-11e8-9f47-a3e5b4bd153e.png)

The empty incidents should get cleaned up when we've got a working `depends` gate. If it's at all possible, I'd also like to add the question on the same line as the date itself so it makes more sense to screen readers. I think we can do that by adding that same title text to the `ui:title` of the date field (and hide it so it doesn't show up on the form page unnecessarily if that's desired).

To add the number to the title, what we've done before in both the pensions and 686 forms is have something like:
```js
const numberToWords = {
  0: 'First',
  1: 'Second',
  2: 'Third',
  3: 'Fourth',
  4: 'Fifth',
  5: 'Sixth',
  6: 'Seventh',
  7: 'Eighth',
  8: 'Ninth',
  9: 'Tenth',
};
```
With that, we can do:
```js
...{
  [`incidentDate${index}`]: {
    title: `${numberToWords[index]} PTSD Incident Date`,
    ...
  }
...
}
```
It's not the most elegant, but unless we have more than 10 iterations, it'll work. :man_shrugging:

Reply:
Changed to match this. This is definitely subject to change once we get closer to the end with all of the pages added.